src/shaderlib/project/project.glsl.js

```diff
@@ -33,38 +33,32 @@ uniform float projectionScale;
 uniform vec4 projectionCenter;
 uniform vec3 projectionPixelsPerUnit;
 
-uniform mat4 modelViewMatrix;
+uniform mat4 modelMatrix;
+uniform mat4 viewMatrix;
 uniform mat4 projectionMatrix;
-uniform mat4 projectionMatrixUncentered;
 
 //
 // Scaling offsets
 //
 
```

Comment:
It's unclear to readers why the scalar version of `project_scale` multiplies meters by the `z` component of `projectionPixelsPerUnit`. We should change the name of this function to `project_scale_z` or at least put a comment here saying that "the scalar version of `project_scale` is for scaling the z component only"